lib/ex_aws/instance_meta.ex

```diff
@@ -12,9 +12,13 @@ defmodule ExAws.InstanceMeta do
   @task_role_root "http://169.254.170.2"
 
   def request(config, url) do
-    # Certain solutions like kube2iam will redirect instance meta requests,
-    # we need to follow those redirects.
-    case config.http_client.request(:get, url, "", [], follow_redirect: true) do
+    # Certain solutions like kube2iam will redirect instance meta requests, we need to follow those redirects.
```

Comment:
Code reorg'd slightly to make it easier for me to comment on the additional options.